.github/workflows/ci.yml

```diff
@@ -180,14 +180,14 @@ jobs:
     needs: [build-tarball, build-wheels]
     runs-on: ubuntu-latest
     steps:
-    - name: Download dists
-      uses: actions/download-artifact@v2
+    - name: Make Release
+      uses: aio-libs/create-release@v1.0.0
       with:
-        name: dist
-        path: dist
-    - name: Count dists
-      run: echo "$(ls dist|wc -l) distributions to upload"
-    - name: Publish a Python distribution to PyPI
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        password: ${{ secrets.PYPI_API_TOKEN }}
+        changes_file: CHANGES.rst
+        name: multidict
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        pypi_token: ${{ secrets.PYPI_API_TOKEN }}
+        artifact: dist
+        version_file: multidict/__init__.py
```

Comment:
Since the code is not fetched, this file is not accessible by the action. Why is it needed anyway? They artifacts should have a version by this point. You could extract the version from the artifacts if necessary.

Reply:
Correct. Let me refactor the action

Reply:
I spent about a day trying to fix the release action without success. Got some strange errors.
Let's merge as-is, please feel free to refactor later.
I'm personally ok with reading the version from `__init__.py`; the action needs repo clone to read `CHANGES.rst` content anyway.
I consider this step as "write and don't touch" and prefer investing my spare time in more urgent tasks, e.g. publishing yarl/aiohttp binary wheels for Python 3.10

Thanks for your understanding.

Reply:
Looks like this one succeeded: https://github.com/aio-libs/multidict/actions/runs/1300483727. What sort of errors are you observing? Or are those the action-related ones?
FWIW I agree that it's not blocking for this specific PR. Feel free to mark this conversation as resolved and merge the PR.
If you have a minute, just make sure to document the problems somewhere, and maybe I'll be able to look at those later, after I address some of the aiohttp blockers.